script.module.resolveurl/lib/resolveurl/plugins/realdebrid.py

```diff
@@ -156,8 +159,16 @@ def get_media_url(self, host, media_id, retry=False, cached_only=False):
                     raise ResolverError('Real-Debrid Error: Failed to transfer torrent to/from the cloud')
 
             url = '%s/%s' % (rest_base_url, unrestrict_link_path)
-            data = {'link': media_id}
-            result = self.net.http_POST(url, form_data=data, headers=self.headers).content
+            if not resolve_all:
+                data = {'link': media_id}
+                result = self.net.http_POST(url, form_data=data, headers=self.headers).content
+            else:
+                unresolved_links = []
+                for index, link in enumerate(torrent_info.get('links')):
+                    # remove leading /
+                    filename = re.sub('^/', '', torrent_info.get('files')[index]['path'])
+                    unresolved_links.append({'name': filename, 'link': link})
+                return unresolved_links
```

Comment:
Can you give me an example of this output `unresolved_links`. I dont have RD account to test

Reply:
Also I think this parameter should be called `return_all` instead of `resolve_all`. What do you reckon?

Reply:
Here's some output from when I was testing:

```json
[
  {
    "name": "[SubsPlease] Super Cub - 01 (1080p) [5A27D649].mkv",
    "link": "https://real-debrid.com/d/R73JQ3STRT6QM"
  },
  {
    "name": "[SubsPlease] Super Cub - 02v2 (1080p) [C8A04E42].mkv",
    "link": "https://real-debrid.com/d/6U52YI35TZ7WQ"
  },
  {
    "name": "[SubsPlease] Super Cub - 03 (1080p) [3EA9788D].mkv",
    "link": "https://real-debrid.com/d/S3CDERZEANP4K"
  },
  {
    "name": "[SubsPlease] Super Cub - 04v2 (1080p) [E40353D3].mkv",
    "link": "https://real-debrid.com/d/UCJJHR5GV3WKI"
  },
  {
    "name": "[SubsPlease] Super Cub - 05v2 (1080p) [23F1E9D7].mkv",
    "link": "https://real-debrid.com/d/RLAE75I75YN3A"
  },
  {
    "name": "[SubsPlease] Super Cub - 06v2 (1080p) [503519EA].mkv",
    "link": "https://real-debrid.com/d/MEEC6FZHD4NKC"
  },
  {
    "name": "[SubsPlease] Super Cub - 07v2 (1080p) [C3AA9550].mkv",
    "link": "https://real-debrid.com/d/ISPEZXRNSDVOQ"
  },
  {
    "name": "[SubsPlease] Super Cub - 08 (1080p) [10FE502E].mkv",
    "link": "https://real-debrid.com/d/PTTANZ2GRUZV4"
  },
  {
    "name": "[SubsPlease] Super Cub - 09 (1080p) [093174A3].mkv",
    "link": "https://real-debrid.com/d/3SLR2CVNRTRFY"
  },
  {
    "name": "[SubsPlease] Super Cub - 10v2 (1080p) [E2E15D2B].mkv",
    "link": "https://real-debrid.com/d/ACVWGBPJCPHCW"
  },
  {
    "name": "[SubsPlease] Super Cub - 11 (1080p) [CC80B96D].mkv",
    "link": "https://real-debrid.com/d/TUNUP2ZD2SRFK"
  },
  {
    "name": "[SubsPlease] Super Cub - 12 (1080p) [F08B4911].mkv",
    "link": "https://real-debrid.com/d/SWGWQADEJPVBS"
  }
]
```

Reply:
And yeah, `resolve_all` doesn't really fit now since it's not actually resolving anymore.
Some ideas on alternatives:
* return_all
* only_list

Reply:
Excellent and the real-debrid resolver has this
`/(http|https):\/\/(\w+\.)?real-debrid\.com\/d\/[0-9A-Z]{13}/`
so it will resolve when you send the individual link to resolve again.

And as for the parameter name, I think `return_all` or perhaps `all_files` suits better.
If you agree, change it in the PR and commit, and I will merge. Thanks

Reply:
I'll go with `return_all` then.

Reply:
Pushed parameter name change.